modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -391,3 +394,117 @@
              (mt/first-row
                (qp/process-query
                  (mt/native-query {:query "select interval '5 days'"}))))))))
+
+;;;; Server side generated timestamps for :relative-datetime tests follow.
+
+(defn- run-native-query [sql & params]
+  (-> (mt/native-query {:query sql
+                        :params params})
+      qp/process-query))
+
+(defn- session-id! []
+  (-> (run-native-query "select pg_backend_pid()") mt/rows ffirst))
+
+(defn- last-queries-of-a-session [session-id]
+  (let [query-str (str "select session_id, start_time, result_cache_hit, query_text\n"
+                       "from sys_query_history\n"
+                       "where session_id = " session-id "\n"
```

Comment:
While writing that I was thinking that here it does not matter much. However, reading your comment I have a feeling it could be a good habit using params by default. I've changed it to param.